### PR TITLE
Adding helper as optional parameter

### LINKED ISSUE #1274: Bug in TwoStepVerificationCommand
After this #1272 pass, I generate a bug, sorry for this!

It's trying to inject null
https://github.com/sonata-project/SonataUserBundle/blob/c4a3fb7bf51d047fff93eb3198bb6d1d91a88e53/src/Resources/config/command.xml#L6

as there isn't **GoogleAuthenticator** in my dependency
https://github.com/sonata-project/SonataUserBundle/blob/c4a3fb7bf51d047fff93eb3198bb6d1d91a88e53/src/Resources/config/google_authenticator.xml#L4 

this service is invalid
https://github.com/sonata-project/SonataUserBundle/blob/c4a3fb7bf51d047fff93eb3198bb6d1d91a88e53/src/Resources/config/google_authenticator.xml#L7-L13

error
```
Argument 2 passed to Sonata\UserBundle\Command\TwoStepVerificationCommand::__construct() must be an instance of Sonata\User  
  Bundle\GoogleAuthenticator\Helper, null given, called in /usr/src/app/var/cache/dev/ContainerG2UYBkm/srcApp_KernelDevDebugC  
  ontainer.php on line 1438                                                                                                    
                                 
```
But as Helper is require, how can I resolve this :confused:

### PR DESCRIPTION
**Pull request by eerison**
## Subject

I am targeting this branch, because this bug is in master

Closes #1274 .

## Changelog

```markdown
### Fixed
- Added Helper as nullable in `TwoStepVerificationCommand`
```

### CODE REVIEW

File: src/Command/TwoStepVerificationCommand.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ final class TwoStepVerificationCommand extends Command
     protected static $defaultName = 'sonata:user:two-step-verification';
 
     /**
-     * @var Helper
+     * @var ?Helper
```
Comment (VincentLanglet) [inline src/Command/TwoStepVerificationCommand.php:30]:
```suggestion
     * @var Helper|null
```

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
@eerison Can you rebase, then I'll merge it.

**Comment by VincentLanglet** [comment]
Oh, we first need to merge 4.x into master

**Comment by eerison** [comment]
> Oh, we first need to merge 4.x into master

I think so, because I tried to update my branch from `master`, But there isn't any change 😕

**Comment by VincentLanglet** [comment]
> > Oh, we first need to merge 4.x into master
> 
> I think so, because I tried to update my branch from `master`, But there isn't any change 😕

Will try to do it tomorrow

**Review by core23 — APPROVED** [review]

**Comment by VincentLanglet** [comment]
@eerison You could be able to rebase now: https://github.com/sonata-project/SonataUserBundle/pull/1277

**Comment by eerison** [comment]
> @eerison You could be able to rebase now: #1277

done :)

**Comment by VincentLanglet** [comment]
Thanks

**Pull request merged**